package.json

```diff
@@ -21,42 +21,40 @@
   },
   "scripts": {
     "build": "yarn build:components",
-    "build:all": "run-s build:preview build:site",
-    "postbuild:all": "yarn mv:preview",
-    "build:clean": "run-s clean build",
-    "build:components": "lerna run --scope \"${SCOPE:-@spectrum-css/*}\" --ignore \"@spectrum-css/{*-builder*,preview,generator}\" build",
-    "postbuild:components": "yarn docs:mod",
-    "build:preview": "nx build @spectrum-css/preview",
-    "build:site": "gulp buildDocs -LLL",
-    "cache:clean": "nx reset",
+    "build:all": "run-s build:preview build:site mv:preview",
+    "build:components": "nx run-many --all --exclude \"@spectrum-css/{*-builder*,preview,generator}\" --target build",
+    "postbuild:components": "node tasks/mod-extractor.js && prettier --cache --write components/*/metadata/*.md > /dev/null",
+    "build:preview": "nx run @spectrum-css/preview:build",
+    "build:site": "gulp build -LLL",
```

Comment:
Just to confirm: this command would not be run on its own, and instead, it'd be part of series where it'd run after a full component build, right?

I ask because running `yarn build:site` directly after a `yarn clean` results in an error, but maybe that was my ignorance in running it as its own command.

Running `yarn build:components` and then `yarn build:site` is successful.

Reply:
Yes, that's true for now. When build:site points to 11ty and it has a package of it's own, we can define build dependencies the same as the storybook site but for now, this would be run as part of the other steps - build:components, build:site, and optionally build:preview